pkg/minibroker/mariadb.go

```diff
@@ -34,58 +45,41 @@ func (p MariadbProvider) Bind(services []corev1.Service, params map[string]inter
 
 	host := buildHostFromService(service)
 
-	dbParams, ok := params["db"].(map[string]interface{})
-	if !ok {
-		dbParams = make(map[string]interface{})
-	}
+	database := provisionParams.DigStringOr("db.name", "")
+	user := provisionParams.DigStringOr("db.user", rootMariadbUsername)
```

Comment:
Previously it was possible to override things in the bind params (e.g. if you created a separate user somehow, you can bind the service to a different app with the new credentials).  That seems like it could be useful?

Reply:
I will create an issue to follow up on this. It deserves a proper mechanism with documentation.

Reply:
https://github.com/kubernetes-sigs/minibroker/issues/101